client/src/components/molecules/ClientRequestTable.tsx

```diff
@@ -0,0 +1,229 @@
+    props.onChangeRequests!(requestsCopy);
+    mutateDeleteRequest({ variables: { id: id } });
+  };
+
+  const onFulfilledRequest = (index: number) => {
+    const requestsCopy = requests.slice();
+    const req = { ...requestsCopy[index] };
+    if (req.fulfilled === false) {
+      req.fulfilled = true;
+      requestsCopy.splice(index, 1);
+      let i = requestsCopy.length - 1;
+      for (; i > -1; --i) {
+        if (requestsCopy[i].fulfilled === false) break;
+        else if (
+          requestsCopy[i]!.dateCreated!.valueOf() < req!.dateCreated!.valueOf()
+        )
+          break;
+      }
+      requestsCopy.splice(i + 1, 0, req);
+    } else {
+      req.fulfilled = false;
+      requestsCopy.splice(index, 1);
+      let i = 0;
+      for (; i < requestsCopy.length; ++i) {
+        if (requestsCopy[i].fulfilled === true) break;
+        else if (
+          requestsCopy[i]!.dateCreated!.valueOf() > req!.dateCreated!.valueOf()
+        )
+          break;
+      }
+      requestsCopy.splice(i, 0, req);
+    }
+    setRequests(requestsCopy);
+    props.onChangeRequests!(requestsCopy);
+    const id = req._id;
+    const requestId = req.requestId;
+    const fulfilled = req.fulfilled;
+    mutateRequest({ variables: { request: { id, requestId, fulfilled } } });
+  };
+
+  return (
+    <div className="request-list">
+      {requestSelectedForEditing && (
+        <RequestForm
+          onSubmitComplete={() => {
+            window.location.reload();
+          }}
+          handleClose={() => setRequestSelectedForEditing("")}
+          operation="edit"
+          requestId={requestSelectedForEditing}
+        />
+      )}
+      {requests.length > 0 && (
+        <Table responsive className="request-table">
+          <thead>
+            <tr>
+              {headingList.map((heading, index) => (
+                <th key={index} className="request-table th">
+                  {heading}
+                </th>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {requests.map((request, index) =>
+              request.deleted === false ? (
+                <tr key={request._id}>
+                  <td>
+                    <div>
+                      <Form.Check
+                        type="checkbox"
+                        onClick={() => onFulfilledRequest(index)}
+                        defaultChecked={request.fulfilled}
+                      />
+                    </div>
+                  </td>
+                  <td
+                    style={
+                      requests[index].fulfilled ? { opacity: 0.2 } : undefined
+                    }
+                  >
+                    <div className="row-text-style">
+                      {request.requestType!.requestGroup!.name}
+                    </div>
+                  </td>
+                  <td
+                    style={
+                      requests[index].fulfilled ? { opacity: 0.2 } : undefined
```

Comment:
Instead of repeating the style, put a class, on the parent (i.e. <tr>)  of all the elements using this common style, when requests[index].fulfilled.

Reply:
done!